solr/solr-ref-guide/java-examples.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Compiling, Testing and Validation for the java examples in the Solr Ref Guide
+
+apply plugin: 'java'
+
```

Comment:
Otherwise this would add a non-existing "main/src" directory to eclipse and other IDEs.

Reply:
I think if "src/main" does not exist it will execute the tasks (including forbiddenapis,...), but all will execute with result "NO-SOURCE".
So I think its fine, unless somebody adds a src/main folder.

Reply:
See issue here: https://discuss.gradle.org/t/impossible-to-remove-main-source-set-when-using-eclipse-plugin/1177

Reply:
I think the same is described in above discussion. So at moment, theres no way to disable the main sourceset. If the directories do not exist in the refguide folder, I think we're fine. It is just stupid that all those tasks are generated and executed.

Reply:
Ahh sorry, somehow I missed one of your comments.